### PR TITLE
Add initial Verona web page.

### PR DESCRIPTION
**Pull request by davidchisnall**
This should be automatically rendered by GitHub Pages at:
https://microsoft.github.io/verona

You can test it locally by installing the github-pages gem and running
`jekyll serve` in the docs directory.

### CODE REVIEW

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <div id="footer_wrap" class="outer">
+      <footer class="inner">
+        {% if site.github.is_project_page %}
+        <p class="copyright">{{ site.title | default: site.github.repository_name }} maintained by <a href="{{ site.github.owner_url }}">{{ site.github.owner_name }}</a></p>
+        {% endif %}
+        <p>Published with <a href="https://pages.github.com">GitHub Pages</a></p>
+      </footer>
+    </div>
+
+    {% if site.google_analytics %}
```
Comment (rengolin) [inline docs/_layouts/default.html:53]: Will we ever use this?

**Review by rengolin — COMMENTED** [review]
We probably also want to change the readme file to point to this page, as well as remove the internal links it has to the same pages.

**Review by mjp41 — APPROVED** [review]
Looks good as far as I can tell.

File: docs/_layouts/default.html (hunk 1 above)
Comment (davidchisnall) [reply docs/_layouts/default.html:53]: No, I just wanted to minimise diffs with the upstream version of this file.

**Pull request merged**